### PR TITLE
PPP service added

### PR DESCRIPTION
**Pull request by n13**
Adding code library for accessing PPP service.

Posted from JavaScript

Uses AWS Amplify and 3 pug ins.

### CODE REVIEW

**Comment by gguijarro-c-chwy** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>:exclamation: PR has more than 500 line changes </td>
    </tr>
  </tbody>
</table>



> Pull Request size seems relatively large. If Pull Request contains multiple changes, split each into separate PR will helps faster, easier review.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 66c2b20004cdda4cbf47826f3c66963a92981161
</p>

**Comment by n13** [comment]
> Fails
> 🚫	❗ PR has more than 500 line changes
> > Pull Request size seems relatively large. If Pull Request contains multiple changes, split each into separate PR will helps faster, easier review.
> 
> Generated by 🚫 [dangerJS](https://danger.systems/js) against [6611047](https://github.com/hypha-dao/hypha_wallet/commit/6611047de78c32881893dde924cab8d5c4fd7a6f)

Guess I could delete all the comments 😄

File: lib/core/ppp_service/ppp.dart [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+  /// @returns {ProfileApi}
+  static ProfileApi profileApi(String activeUser) {
+    instance.checkUser(activeUser);
+    instance._profileApi ??= ProfileApi();
+    return instance._profileApi!;
+  }
+
+  // TODO(n13): Put json in file once it's working
+  static const Map<String, dynamic> ppp_config_map = {
```
Comment (Alex-IS) [inline lib/core/ppp_service/ppp.dart:65]: @n13 You've wrote it already, but just to confirm. Config details should be retrieved from environment variables

**Comment by n13** [comment]
It's missing amplifyconfiguration.dart - working with Sebastian to get that

**Comment by n13** [comment]
This is not working, have not yet been able to get a proper 

amplifyconfiguration.dart

It's extremely complicated and likely we will only solve this when we take over profile service, and remove it from the claws of AWS. 

I made a temporary access server so we can access public profile data easily, will use this for the time being in read only mode, until we find a permanent solution for both DHO and wallet.

**Comment by n13** [comment]
closing this

**Pull request closed without merging**